api/problem_details.py

```diff
@@ -57,7 +57,7 @@
 
 NO_ACCEPTABLE_FORMAT = pd(
```

Comment:
The status codes here were changed because the only place that used these `ProblemDetails` was the loans controller exception handler, where they were called like this:
```
CANNOT_FULFILL.with_debug(str(e), status_code=e.status_code)
```

So they always had the status code of the exception, rather then the status code associated with the `ProblemDetail`. This let me drop the status code on the exceptions and just use the one on the ProblemDetail.